docs/openstack/edpm_adoption.md

```diff
@@ -177,6 +177,45 @@ lands.
   ``cell<X>``. That secret, alongside ``nova-migration-ssh-key``, should
   always be specified for each custom `OpenStackDataPlaneService` related to Nova.
 
+* Create a repo-setup service to configure Antelope repositories
+
+  ```yaml
+  oc apply -f - <<EOF
+  apiVersion: dataplane.openstack.org/v1beta1
+  kind: OpenStackDataPlaneService
+  metadata:
+    name: repo-setup
+    namespace: openstack
+  spec:
+    label: dataplane.deployment.repo.setup
+    play: |
+      - hosts: all
+        strategy: linear
+        tasks:
+          - name: Enable podified-repos
+            become: true
+            ansible.builtin.shell: |
+              # TODO: Use subscription-manager and a valid OSP18 repos instead
+              # This is a hack to deploy RDO Delorean repos to RHEL as if it were Centos 9 Stream
+              set -euxo pipefail
+              curl -sL https://github.com/openstack-k8s-operators/repo-setup/archive/refs/heads/main.tar.gz | tar -xz
+              python3 -m venv ./venv
+              PBR_VERSION=0.0.0 ./venv/bin/pip install ./repo-setup-main
+              # This is required for FIPS enabled until trunk.rdoproject.org
+              # is not being served from a centos7 host, tracked by
+              # https://issues.redhat.com/browse/RHOSZUUL-1517
+              dnf -y install crypto-policies
+              update-crypto-policies --set FIPS:NO-ENFORCE-EMS
+              # FIXME: perform dnf upgrade for other packages in EDPM ansible
+              # here we only ensuring that decontainerized libvirt can start
+              ./venv/bin/repo-setup current-podified -b antelope -d centos9 --stream
+              dnf -y upgrade openstack-selinux
```

Comment:
I wonder if over longer term we should be doing RPM updates in a different role, and this one should be reserved for repository file setup only... But anyway +1 on the unblock and thanks :).

Reply:
as you know, there is a related work for RPM updates in EDPM https://github.com/openstack-k8s-operators/edpm-ansible/pull/532 . If it lands sooner than later, we could build on top of it

Reply:
+1 thanks. (It's not a blocker either way.)

Reply:
so based on other slack discussion i have drop al the package management changes out of
https://github.com/openstack-k8s-operators/edpm-ansible/pull/532
so we should just proceed with this pr to unblock adoption